### DISCUSSION TITLE
Fetch replies to my account via the API

### DISCUSSION THREAD

**Discussion by rmdes** [bluesky-social/atproto#1484] [How to do it?] [answered]
I want to learn how to fetch replies to my account using a python/bash-curl script and then forward the JSON object to my ntfy.sh mobile app so to have basic notifications, can anyone point out where I should start to have the basic running ?

I have glanced over here : https://github.com/MarshalX/atproto/tree/main/examples
but I can't find an example where a reply to an account is showed

Maybe this is the best approach to get this : https://github.com/MarshalX/atproto/blob/main/examples/advanced_usage/notifications_callback.py

Along with this ressource ? https://atproto.com/lexicons/app-bsky-notification#appbskynotificationlistnotifications

**Comment by bnewbold** [comment, marked as answer]
Hi @rmdes, if you are looking for a Python-specific solution using MarshalX's library, you should probably ask over in the https://github.com/MarshalX/atproto repository directly.

In general, yes, the `app.bsky.notification.listNotifications` endpoint on the PDS is used to get all the various kinds of notifications for the account. It requires auth, and can be paginated.

**Reply by rmdes** [reply]
thanks very much, yes I have already managed to get the data using curl/bash approach, it works great!

will adapt now to python!
